onnxruntime/python/tools/quantization/base_quantizer.py

```diff
@@ -230,7 +231,14 @@ def quantize_bias_static_impl(self, bias_name, input_scale, weight_scale, beta=1
             # TODO: This formula should be explained including why the scale is not estimated for the bias as well.
             bias_scale = input_scale * weight_scale * beta
 
-            quantized_data = (np.asarray(bias_data) / bias_scale).round().astype(np.int32)
+            try:
+                with warnings.catch_warnings():
+                    warnings.filterwarnings('error')
+                    quantized_data = (np.asarray(bias_data) / bias_scale).round().astype(np.int32)
+            except RuntimeWarning:
```

Comment:
Does it work to catch a warning like that?

Reply:
I would probably make a test checking the min and max before rounding instead of catching an exception.

Reply:
Checking the min/max instead of catching an exception.